src/components/Navbar/index.spec.tsx

```diff
@@ -23,18 +23,23 @@ import Navbar from '.';
 describe(__filename, () => {
   type RenderParams = {
     _requestLogOut?: typeof requestLogOut;
+    reviewersHost?: string;
     store?: Store;
   };
 
   const render = ({
     _requestLogOut = jest.fn(),
+    reviewersHost,
```

Comment:
Since this is optional, you could pass it in as `...props`. That would also make this `render()` more consistent with most others.